Author: Douglas Jerome <djerome@crosslinux.net>
Date:   Sat Mar 15 15:19:37 MST 2014

        This patch changes a function name to avoid a system function that may
        have the same name.

        Signed-off-by: Douglas Jerome <djerome@crosslinux.net>

diff -Naur thttpd-2.25b-OLD/extras/htpasswd.c thttpd-2.25b-NEW/extras/htpasswd.c
--- thttpd-2.25b-OLD/extras/htpasswd.c	2010-04-30 18:49:55.000000000 -0700
+++ thttpd-2.25b-NEW/extras/htpasswd.c	2014-03-15 12:08:08.000000000 -0700
@@ -49,7 +49,7 @@
     while((line[y++] = line[x++]));
 }
 
-static int getline(char *s, int n, FILE *f) {
+static int thttpd_getline(char *s, int n, FILE *f) {
     register int i=0;
 
     while(1) {
@@ -189,7 +189,7 @@
     strcpy(user,argv[2]);
 
     found = 0;
-    while(!(getline(line,MAX_STRING_LEN,f))) {
+    while(!(thttpd_getline(line,MAX_STRING_LEN,f))) {
         if(found || (line[0] == '#') || (!line[0])) {
             putline(tfp,line);
             continue;
